doc/nrf/releases_and_maturity/software_maturity.rst

```diff
@@ -3046,6 +3069,7 @@ The following table indicates the software maturity levels of the support for ea
               - --
             * - **Image encryption**
               - Experimental
+
```

Comment:
LM20 MCUboot is <B>experimental</B> in features range of:
Immutable MCUboot as part of build,
Application image compressio,
Application image encryption.

Reply:
What about Updatable MCUboot as part of build?